exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -96,17 +100,30 @@ def _translate_key_values(key: Text, value: Any) -> KeyValue:
         any_value = AnyValue(double_value=value)
 
     elif isinstance(value, Sequence):
-        any_value = AnyValue(array_value=value)
+        any_value = AnyValue(
+            array_value=ArrayValue(values=[_translate_value(v) for v in value])
+        )
 
-    elif isinstance(value, Mapping):
-        any_value = AnyValue(kvlist_value=value)
+    # Tracing specs currently does not support Mapping type attributes
+    # elif isinstance(value, Mapping):
+    #     any_value = AnyValue(
+    #         kvlist_value=KeyValueList(
+    #             values=[
+    #                 _translate_key_values(str(k), v) for k, v in value.items()
+    #             ]
+    #         )
+    #     )
```

Comment:
Does this mean any spans containing a Mapping attribute will throw an exception below? I don't think we should do that if the API allows to create KV attributes (not sure if it does). May be we can warn and drop the attribute instead?

Reply:
API does not allow adding an attribute that is of type Mapping (https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L67)